.eslintrc.js

```diff
@@ -48,11 +48,12 @@ module.exports = {
      * ESLint and Airbnb related rules
      * https://eslint.org/docs/rules/
      */
+    'class-methods-use-this': 'off', // be able to define functions, that no using this, that can be used by vue templates
```

Comment:
If you want to use a function, that does not uses any this context within a vue template, it must be declared within the class. Or am i wrong?

Like:

```ts
  isCorrectStatement(paramA, paramB) {
    return paramA && paramB
  }
```

```vue
<div v-if="isCorrectStatement(true, false)"> ...
```

Reply:
Yes, functions must be declared in the class. You simply declare them as static functions then.

```
static isCorrectStatement(paramA, paramB) {
    return paramA && paramB
  }
```

Reply:
Yes it doesn't work :D

The isCorrectStatement is bound to the class definition and not to the instance. The vue context is only bound to instance and will throw, because it's not defined on the instance.